libflux/src/flux/semantic/analyze.rs

```diff
@@ -262,7 +262,13 @@ fn analyze_function_params(
 
 fn analyze_function_body(body: ast::FunctionBody, fresher: &mut Fresher) -> Result<Block> {
     match body {
-        ast::FunctionBody::Expr(e) => Ok(Block::Return(analyze_expression(e, fresher)?)),
+        ast::FunctionBody::Expr(expr) => {
```

Comment:
Yeah, I did pull and rebase after merging those changes. I can do it again for safekeeping.

Reply:
I also rebased an earlier version of these changes onto this feature branch a few commits ago so that I could start using those changes on this branch. Maybe that's why its showing up like this.